neofetch

```diff
@@ -84,6 +84,12 @@ get_distro() {
                     *) distro="Siduction ($(lsb_release -sic))"
                 esac
 
+            elif [[ -x "/usr/bin/pveversion" ]]; then
+                case "$distro_shorthand" in
+                    "on" | "tiny") distro="Proxmox VE" ;;
+                    *) distro="Proxmox VE $(pveversion | grep -oP 'pve-manager\/\K\d+\.\d+')"
```

Comment:
What's the output of `pveversion` on its own? Grep may not be needed if we can use bash builtins.